src/resourcemgr/fileBrowser.js

```diff
@@ -1,52 +1,77 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2018-2020 (original work) Open Assessment Technologies SA;
+ */
 import $ from 'jquery';
 import _ from 'lodash';
+import paginationComponent from 'ui/pagination';
+import rootFolderTpl from 'ui/resourcemgr/tpl/rootFolder';
+import folderTpl from 'ui/resourcemgr/tpl/folder';
 
-var ns = 'resourcemgr';
+const ns = 'resourcemgr';
 
-export default function(options) {
-    var root = options.root || '/';
-    var path = options.path || '/';
-    var $container = options.$target;
-    var $fileBrowser = $('.file-browser .file-browser-wrapper', $container);
-    var $divContainer = $('.' + root, $fileBrowser);
-    var $folderContainer = $('.folders', $divContainer);
-    var fileTree = {};
+export default function (options) {
+    const root = options.root || '/';
+    const path = options.path || '/';
+    const $container = options.$target;
+    const $fileBrowser = $('.file-browser .file-browser-wrapper', $container);
+    const $divContainer = $('.' + root, $fileBrowser);
```

Comment:
Weird, in case of `options.root` default value  is `/` the selector is going to be `$('./')` 🤔